### ISSUE TITLE
Screen Reader Compatibility on Quote Cards

### ISSUE THREAD

**Issue by Manali-SS** [Paul-Austin-Oswego-CSC480-HCI521/OZ-CSC-480-HCI-521-Spring-2025#110]
Reference ID: LOG_065

Priority:
Medium

Execution Steps:
Use the screen reader to navigate through the Quote Cards.
Ensure all elements have meaningful labels and roles.

Problem:
There are no labels for any buttons. The screen reader reads it as a button.

Expected Results:
All interactive elements are read correctly.
Alt text is present for icons and buttons.

Evidence:
[Video link](https://drive.google.com/file/d/1VICaWRLzu0yfKaVX_uZhW8FzvGwrQs9Q/view?usp=sharing)

**Comment by Manali-SS** [comment]
@gob1ingo, this is partially completed. Raising a new issue!

**Comment by gob1ingo** [comment]
What is the issue?

**Comment by Manali-SS** [comment]
@gob1ingo There are no labels for the share and report buttons when using a screen reader.

**Comment by gob1ingo** [comment]
Should now be solved by pull request #222
